### ISSUE TITLE
Oasis CLI is lacking man pages

### ISSUE THREAD

**Issue by omitar** [oasislabs/oasis-rs#202]
Running `man oasis` after Oasis CLI has been installed:

```
No manual entry for oasis
See 'man 7 undocumented' for help when manual pages are not available.
```

**Comment by nhynes** [comment]
This is, in general, a hard problem. Note that even golang does not have a man page.

**Comment by omitar** [comment]
Hard problem in which way? How to install them? Or how to generate them? There are probably tools for both?

**Comment by nhynes** [comment]
is `oasis help` not sufficient? generating man pages isn't automatic

**Comment by omitar** [comment]
> is oasis help not sufficient?

Depends on the amount of information you provide. Currently `help` does not provide enough documentation anyway. But if you would really start providing full documentation to the degree `git` does, then `help` would not be really beneficial. I would even say you could do the same as `git`, simply opening man page if you run help.

> generating man pages isn't automatic

There are Markdown to man page converters:

http://rtomayko.github.io/ronn/ronn.1

Probably others could be find as well.

**Comment by nhynes** [comment]
>  Currently help does not provide enough documentation

Yeah I just got that feedback from someone else. I'm hoping to revive #148. Maybe if the format is amenable to man pages, I'll add it (but probably not because managing installs and uninstalls is arduous).

**Comment by nhynes** [comment]
Also, this was filed in the wrong repo.
